signup.html

```diff
@@ -1,36 +1,41 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Signup Page</title>
-    <link rel="stylesheet" href="signup.css" />
-  </head>
-  <body>
-    <header>
-      <nav>
-          <a href="index.html">Signup</a>
-      </nav>
-  </header>
-  <div class="container">
-        <form class="form-container" onsubmit="return validateForm(event)">
-          <h2>Sign Up Form</h2>
-          <div class="form-group">
-               <input type="text" id="username"  placeholder="Enter your username" >
-               <span id="usernameError" class="error"></span>
-          </div>
-          <div class="form-group">
-              <input type="text" id="email"  placeholder="Enter your email" >
-              <span id="emailError" class="error"></span>
-           </div>
-          <div class="form-group">
-              <input type="password" id="password" placeholder="Enter your password" >
-              <span id="passwordError" class="error"></span>
-          </div>
-           <button type="submit">Submit</button>
-           <div id="message" class="message"></div>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="signup.css">
+    <title>HTML Form Validation</title>
```

Comment:
Add a page tittle which is more relevant like Signup

Reply:
solve this